commercetools-models/src/main/java/io/sphere/sdk/orders/commands/OrderFromCartCreateCommandImpl.java

```diff
@@ -25,6 +25,10 @@ public static OrderFromCartCreateCommandImpl of(final Versioned<Cart> cart) {
         return new OrderFromCartCreateCommandImpl(OrderFromCartDraft.of(cart));
     }
 
+    public static OrderFromCartCreateCommandImpl of(final Cart cart) {
+        return new OrderFromCartCreateCommandImpl(OrderFromCartDraft.of(cart));
+    }
```

Comment:
not needed a Cart implements Versioned

Reply:
I'm replacing with ResourceIdentifier<Cart>